### DISCUSSION TITLE
终于当上绳匠了！！！

### DISCUSSION THREAD

**Discussion by qishaming-mihomo** [share121/inter-knot#2302] [其他]
以前从没接触过这些东西，捣鼓俩小时终于玩明白了
祝绳网越来越好

**Comment by KUYUE404** [comment]
哎嘿<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
